### ISSUE TITLE
Why MBOX concentrations showed up in non-NA regions?

### ISSUE THREAD

**Issue by EnyuxPolyU** [geoschem/geos-chem#2384]
### Your name

Enyu Xiong

### Your affiliation

The HongKong Polytechnic University

### Please provide a clear and concise description of your question or discussion topic.

Dear GEOSCHEM support team,

I am trying to add Methylbutenol (MBOX, speceis only in NA) back to GC14.4.1 to enhance the yield of Mgly and Gly, but I found MBOX showed up in other regions. The figure is the quick look by PanolyWin.

![3](https://github.com/user-attachments/assets/4f9b61a7-8108-4ad7-9973-b888defed0db)

Here is the modification for my codes:

1. modified hcox_megan_mod.F90:

```F90
       !--------------------------------------------------------------
       ! Added MEGAN MBOX 0716
       !--------------------------------------------------------------
       IF ( Inst%IDTMBOX > 0 ) THEN
          Inst%FLUXMBOX(I,J) = EMIS_MBOX
       ENDIF    


    !Added MBOX 0716
    ALLOCATE( Inst%FLUXMBOX( NX, NY ), STAT=AS )
    IF ( AS /= 0 ) THEN
       CALL HCO_ERROR( 'FLUXMBOX', RC )
       RETURN
    ENDIF
    Inst%FLUXMBOX = 0.0_hp
```

2. registered MBOX in species_database.yml and custom.eqn (following the tutorial)
```yaml
MBOX:
  Formula: C5H10O
  FullName: Methylbutenol
  MW_g: 88.148
  Is_Gas: true
  Is_Advected: true
```

```console
MBOX + OH = 0.054GLYX + 0.062MGLY：  GCARR_ac(8.10d-12, 610.0d0)；
```
Thank you very much!
Sincerely,
Enyu

**Comment by yantosca** [comment]
Thanks for writing @EnyuxPolyU.  Which version of HEMCO are you using?  The current version 3.9.1 has MBOX included as an emission species (at least for internal use).

**Comment by EnyuxPolyU** [comment]
> Thanks for writing @EnyuxPolyU. Which version of HEMCO are you using? The current version 3.9.1 has MBOX included as an emission species (at least for internal use).

Thank you @yantosca . I am using GC14.4.1 with HEMCO 3.9.1.
Yes, I do find MBOX emission code in MEGAN code. However, when I tried to add MBOX in the MEGAN in HEMCO_Config.rc, it would create error. 


108     MEGAN                  : on    ISOP/ACET/PRPE/C2H4/ALD2/MOH/EOH/MTPA/MTPO/LIMO/SESQ/SOAP/SOAS/MBOX

HEMCO ERROR: Invalid species names: MBOX
HEMCO ERROR: Error encountered in "HCOX Megan Init"!-> LOCATION:→> at HCOX INIT(in module HEMC0/Extensions/hcox driver mod.F90


That's why I tried to add several sections in the code. I will also check the result with my group members, but our server are undering maintainance these days. I wil update the result as quickly as possible.

Thank you very much.

**Comment by yantosca** [comment]
Thanks for the update @EnyuxPolyU,  I believe that you might need to add MBOX to the list of `transported species` in the `geoschem_config.yml` file.  Unless the species is transported I don't think you can assign it an emissions (I could be wrong though).  You would also want to set the time refresh field for the restart file `SPC_` from `EFYO` to `EY` so that it will allow missing values.

**Comment by EnyuxPolyU** [comment]
Dear @yantosca , 
I compared the result with my group member and found that their standard simulation did not have MBOX concentration in non-NA regions. Their result looks fine.

I will just disable MBOX from now because it would not affect my research. Maybe the problem is because I added too many dummy species to track oxidations for budget diagnosis purpose. I will keep an eye on this problem.   

Thank you very much.
